fixtures/demo.json

```diff
@@ -1,714 +1,4 @@
 [
```

Comment:
Has this file been kept deliberately as well as the moved one?

Reply:
I didn't want to delete it as the documentation still references it https://softwaresaved.github.io/lowfat/doc/testing.html
I will update the documentation and we can delete the old?

Reply:
I think the following commits fix that?

Reply:
Nearly! The other deleted files (flatpages and sites) we actually do need to keep - those are part of the deployment process of a production system - but moving them into the same place as you did the demo fixtures would make sense

Reply:
Sorry, that was my ignore file and not paying enough attention they should be back now!